man/cgroup-sandboxing.xml

```diff
@@ -6,6 +6,7 @@
 -->
 
 <refsect1>
+  <title/>
 
 <para id="singular">This option cannot be bypassed by prefixing <literal>+</literal> to the executable path
```

Comment:
Could you adjust indentation here and below?

Reply:
Sure.